### PR TITLE
Add static image

### PR DESCRIPTION
**Pull request by bryancr89**

### CODE REVIEW

File: .env-dev [hunk 1]
```diff
@@ -1,6 +1,7 @@
 APP_DEFAULT_PATH=/dashboard
 APP_DOMAIN=escrowcommons.io
 APP_GOOGLE_MAPS_API_KEY=
+APP_STATIC_GOOGLE_MAPS_API_KEY=
```
Comment (brock-haugen) [inline .env-dev]: We shouldn't need this env variable here -- `APP_GOOGLE_MAPS_API_KEY` would take care of it

File: app/lib/maps.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import getConfig from 'next/config';
+
+const {
+  publicRuntimeConfig: { APP_STATIC_GOOGLE_MAPS_API_KEY },
```
Comment (brock-haugen) [inline app/lib/maps.js]: `APP_GOOGLE_MAPS_API_KEY`

File: app/lib/maps.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+} = getConfig();
+
+const getStaticMap = (meta = {}) => {
+  const { lat, lng, photo } = meta;
+  const url = '/static/images/background.jpg';
+  let googleMapUrl = '';
+
+  if (lat && lng) {
+    googleMapUrl = `https://maps.googleapis.com/maps/api/staticmap?center=${lat},${lng}&zoom=13&scale=2&format=png&size=900x500&key=${APP_STATIC_GOOGLE_MAPS_API_KEY}`;
```
Comment (brock-haugen) [inline app/lib/maps.js]: `APP_GOOGLE_MAPS_API_KEY`

File: app/lib/maps.js (hunk 2 above)
Comment (bryancr89) [reply app/lib/maps.js]: Removed

**Pull request merged**